.github/workflows/curl-collector.yaml

```
@@ -30,5 +30,5 @@ jobs:
         if: ${{ failure() }}
         uses: ./certsuite/.github/actions/slack-webhook-sender
         with:
-          message: 'Collector GET request has failed'
+          message: 'Collector GET request has failed. Please trigger the job manually https://github.com/redhat-best-practices-for-k8s/collector/actions/workflows/collector-image.yaml'
```

```suggestion
          message: >
            Collector GET request has failed.
            Please trigger the job manually
            https://github.com/redhat-best-practices-for-k8s/collector/actions/workflows/collector-image.yaml
```

See more:
https://stackoverflow.com/questions/3790454/how-do-i-break-a-string-in-yaml-over-multiple-lines